client.go

```diff
@@ -335,15 +335,34 @@ func (c *ovndb) reconnect() {
 	}()
 }
 
-func (c *ovndb) MonitorTables(jsonContext interface{}) (*libovsdb.TableUpdates, error) {
-	// get the table list based on the DB
+// checkSchemaTables checks whether tables in
+// NBTablesOrder / SBTablesOrder exists in current ovn-db schema
+func (c *ovndb) checkSchemaTables() []string {
+	schemaTables := make([]string, 0)
 	var tables []string
+
+	// get the table list based on the DB
 	if c.db == DBNB {
 		tables = NBTablesOrder
 	} else {
 		tables = SBTablesOrder
 	}
 
+	dbSchema, err := libovsdb.GetSchema(c.db)
```

Comment:
Did you test this in your local as CI is failing?
```
./client.go:351:19: undefined: libovsdb.GetSchema
note: module requires Go 1.12FAIL	github.com/ebay/go-ovn [build failed]
```

Seems we need to handle such error in CI too to avoid showing exited with 0. instead of failure.

Reply:
@noah8713 updated it.